src/pybind/mgr/nfs/export.py

```diff
@@ -267,42 +282,43 @@ def _delete_export_user(self, export: Export) -> None:
             # do nothing; we're using the bucket owner creds.
             pass
 
-    def _create_export_user(self, export: Export) -> None:
-        if isinstance(export.fsal, CephFSFSAL):
-            fsal = cast(CephFSFSAL, export.fsal)
-            assert fsal.fs_name
-            fsal.user_id = f"nfs.{export.cluster_id}.{export.export_id}"
-            fsal.cephx_key = self._create_user_key(
-                export.cluster_id, fsal.user_id, export.path, fsal.fs_name
+    def _create_rgw_export_user(self, export: Export) -> None:
```

Comment:
this is touching RGW code, I guess the RGW team should be consulted to make sure it doesn't change anything unknowingly.

Reply:
I dont think anything's changed here I've just changed function name to handle cephfs & rgw things seperately

Reply:
yeah, i hope there are no changes since i'm not quite equipped with rgw code to validate it :)

Reply:
I think the unit tests are passing for rgw nfs already which proves nothing's breaking!